pkg/apiserver/info/info.go

```diff
@@ -38,15 +38,16 @@ func NewService(config *config.Config, tidbForwarder *tidb.Forwarder, db *dbstor
 
 func Register(r *gin.RouterGroup, auth *user.AuthService, s *Service) {
 	endpoint := r.Group("/info")
-	endpoint.Use(auth.MWAuthRequired())
 	endpoint.GET("/info", s.infoHandler)
+	endpoint.Use(auth.MWAuthRequired())
 	endpoint.GET("/whoami", s.whoamiHandler)
 	endpoint.GET("/databases", utils.MWConnectTiDB(s.tidbForwarder), s.databasesHandler)
```

Comment:
For the /databases endpoint, it requires `MWAuthRequired`. However I guess this API is not used anywhere. You can verify it (and remove it)

Reply:
Refine it by PR #646 , use this API to replace the `/statements/schemas` API.